modules/getting-started/proc-audit-log-view.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// assembly-audit-log.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-audit-log-view_{context}"]
+= Viewing audit logs in {product-short}
+
+Administrators can view, search, filter, and manage the log data from the {ocp-brand-name} web console. You can filter audit logs from other log types by using the `isAuditLog` field.
+
+.Prerequisites
+* You are logged in as an administrator in the {ocp-short} web console.
+
+.Procedure
+
+. From the *Developer* perspective of the {ocp-short} web console, click the *Topology* tab.
+. From the *Topology* view, click the *Resources* tab.
```

Comment:
Before clicking on the Resources Tab, the user needs to click on the rhdh-backstage pod.

Reply:
@gustavolira thank you. Is the name of the pod generic or specific / the same every time? Can we say something like _Click on the pod that you want to view audit log data for_? Or do we need to specifically say _Click on the `rhdh-backstage` pod_?

Reply:
The name is generic, it can be anything. So we can say to click in the backstage /RHDH pod, or something like that